doc/coog_admin_tuto.rst

```diff
@@ -0,0 +1,676 @@
+    ./upgrade \
+        -t <image-tag> \
+        -a <image-archive> \
+        -p <image-repository> \
+        -s <server-workers-number> \
+        -c <celery-workers-number> \
+        -b : backup database \
+        -u : update database \
+        -h : print upgrade command help
+
+Backup procedure
+----------------
+
+In order to regularly keep database and attachments backups, coog-admin offers
+a backup command.
+
+In order to execute the backup command, create a backup directory. By default,
+the backup directory is set to
+
+*/mnt/coog_backup*
+
+Execute
+
+.. code-block:: bash
+
+    ./conf edit
+
+Edit the environment variable *BACKUP_DIRECTORY* with the path to this
+directory.
+
+In order to delete daily backups of more than seven days, run the command:
+
+.. code-block:: bash
+
+    ./backup clean
+
+In order to launch the backup command, you have to be in your *coog-admin*
+directory. When you are in, launch the following command:
+
+.. code-block:: bash
+
+    ./backup save
+
+This will generate an archive for the database and another one for attachments
+in *$BACKUP_DIRECTORY*.
+
+This command also does an additional backup on
+
+* The first day of the year
+* The first day of the month
+* The first day of the week
+
+Both commands (clean and save) can be programmed in a *crontab* to be
+automatically launched everyday. In order to do so, edit the user's
+*crontab* using the comand:
+
+.. code-block:: bash
+
+    crontab -e
+
+Add the following lines:
+
+.. code-block:: bash
+
+    <min> <h> * * * USER=<username> DB_NAME=<db_name> COOG_DATA=<path_to_data> \
+        <path/to/coog-admin/>/backup clean
+    <min> <h> * * * USER=<username> DB_NAME=<db_name> COOG_DATA=<path_to_data> \
+        <path/to/coog-admin/>/backup save
+
+More about coog-admin commands
+------------------------------
+
+If you want to know more about coog-admin scripts and the possibilities you
+have, just run the script with no arguments, they are all self documented
+(./coog ./redis )
+
+Here are some useful commands:
+
+.. code-block:: bash
+
+    ./coog reset
+    ./coog version # gives the repositories list and the last commits
+    ./coog conf # displays workers configuration for app and batch
+    ./coog env # displays environment variables for coog containers
+    ./coog module list # displays coog installed modules list
+    ./coog admin -u <modules separated by commas> # installs/ updates modules
+                                                  # list
+    ./coog server [nb-workers] # launches application workers
+    ./coog celery [nb-workers] # launches batch workers
+
+To obtain logs:
+
+.. code-block:: bash
+
+    ./coog -- server logs
+    ./coog -- celery logs
+ 
+Sentry
+------
+
+Create a new database named *sentry*
+After that, run the following command
+
+.. code-block:: bash
+
+    ./sentry upgrade
+
+Create an account
+
+.. code-block:: bash
+
+    ./sentry server
+    ./sentry cron
+    ./sentry worker
+
+Connect to localhost:9000
```

Comment:
The server does not often has a graphical env for us to use a web browser. I'd say this should by server-ip:9000 , no ?